interface and embedded struct build error

<!-- Please make sure to run `v up` before reporting any issues as it may have already been fixed.
     It's also advisable to update all relevant modules using `v outdated` and `v install` -->

<!-- You can use `v doctor` to fill up the next fields -->
**V version: 0.2.4 3fa6622** 
**OS: ArchLinux** 

<!-- Please include the standalone minimal reproducible code.
If not, it may be closed by the administrators. -->
**What did you do?**
```v
pub struct ViewBase{}

pub fn (vb ViewBase) point_inside(i int){}

//----

pub struct ContainerBase{
	ViewBase
}

//----

interface Focusable{
	point_inside(int)
}

```

**What did you expect to see?**
No error
 
**What did you see instead?**
```
> v run bug.v
==================
/tmp/v_1000/bug.10568630471265921366.tmp.c:1934: error: field not found: ViewBasee
...
==================
```

--- a/vlib/v/gen/c/cgen.v
+++ b/vlib/v/gen/c/cgen.v
@@ -5633,6 +5633,9 @@ static inline __shared__$interface_name ${shared_fn_name}(__shared__$cctype* x) 
 								methods_wrapper.write_string('.$esym.embed_name()')
 							}
 						}
+						if fargs.len > 1 {
+							methods_wrapper.write_string(', ')
+						}
 						methods_wrapper.writeln('${fargs[1..].join(', ')});')
 					} else {
 						if parameter_name.starts_with('__shared__') {

--- a/vlib/v/tests/interface_and_embedded_struct_build_test.v
+++ b/vlib/v/tests/interface_and_embedded_struct_build_test.v
@@ -1,0 +1,17 @@
+pub struct Event {}
+
+pub struct ViewBase {}
+
+pub fn (vb ViewBase) point_inside(e Event) {}
+
+pub struct ContainerBase {
+	ViewBase
+}
+
+interface Focusable {
+	point_inside(Event)
+}
+
+fn test_interface_and_embedded_struct_build() {
+	assert true
+}

